src/OpenLoco/GameCommands/UpdateOwnerStatus.cpp

```diff
@@ -0,0 +1,29 @@
+#include "../CompanyManager.h"
+#include "GameCommands.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    // 0x004383CA
+    static uint32_t updateOwnerStatus(const uint8_t flags, OwnerStatus ownerStatus)
```

Comment:
this cant be const &

Reply:
I assume you meant can and not can't, so I changed `OwnerStatus` to `const OwnerStatus&`